superset/initialization/__init__.py

```diff
@@ -563,6 +563,8 @@ def init_app(self) -> None:
         # Configuration of feature_flags must be done first to allow init features
         # conditionally
         self.configure_feature_flags()
+        if feature_flag_manager.is_feature_enabled("SECRET_KEYS_ROTATIONS"):
```

Comment:
Just always call `self.configure_secret_keys()`, which performs a no-op by default. Then, downstreams can implement their own logic by extending `Supersetinitializer`

Reply:
by no-op you mean a default empty implementation the does nothing?

Reply:
Yep.